test/integration/suites/ackekscluster/generator.go

```diff
@@ -43,8 +43,8 @@ func eksCluster(
 				"clusterARN": "${cluster.status.ackResourceMetadata.arn}",
 			},
 		),
+		generator.WithResource("clusterRole", clusterRoleDef(namespace), nil, nil),
```

Comment:
I figured out that it is much easier to reorder the objects in the "manifest" than it is to reorder the mocking behaviour :-)

Reply:
clever, looove it :)